server/tasks/index.ts

```
@@ -0,0 +1,40 @@
+  private static _instance: TaskManager;
+  private inngest: Inngest;
+  public jobs: any = [];
+
+  private constructor() {
+    // TODO: each job could be dynamically loaded from a directory here
+    // and defined in the setupAgenda method
+    this.inngest = new Inngest({ id: config.inngest.id });
+  }
+
+  public static getInstance(): TaskManager {
+    if (!this._instance) {
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)